.teamcity/settings.kts

```diff
@@ -295,6 +301,93 @@ object MyPy : BuildType({
     }
 })
 
+object PipPython310Template : Template({
+    name = "PipPython310Template"
+
+    detectHangingBuilds = false
+
+    vcs {
+        root(DslContext.settingsRoot, "+:. => imod-python")
+
+        cleanCheckout = true
+    }
+
+    steps {
+        script {
+            name = "Pip install python 3.10"
+            id = "pip_install_py310"
+            workingDir = "imod-python"
+            scriptContent = """
+                    pixi info
```

Comment:
pixi info is no longer needed now that you have debugged and fixed the pipeline script

Reply:
True, removed this command.